portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,325 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css?family=Source+Code+Pro');
+*{
+  margin: 0;
+  padding: 0;
+
+  
+  /*font-family: sans-serif;
+  letter-spacing: 1px;
+  font-weight: 300;*/
+}
+h2{
+  text-align: center;
+}
+h1, h2, h3{
+  color: green;
+  font-size: 25px; 
+  
+}
+/*body{
+  overflow-x: hidden;
+}*/
+nav{
+  color: #f2f5f7;
+  height: 4rem;
+  width: 100vw;
```

Comment:
Google's CSS style guide recommends alphabetizing your CSS declarations: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

I actually don't love this rule myself, so as with other style guide things it's totally up to you whether you want to follow it or not!

Reply:
Amazing, i checked the link, i will change as well, thanks